### PR TITLE
@craigspaeth => Options to not grow images past desired height

### PR DESCRIPTION
**Pull request by mzikherman**
if you pass `false`, then if the images already fit they won't get resized up.

Needed for https://github.com/artsy/force/issues/5213

### CODE REVIEW

File: index.coffee [hunk 1]
```diff
@@ -41,9 +42,13 @@ fillwidth = ($list, targetHeight, done, apply, gutterSize) ->
       img.width = img.width * (targetHeight / img.height)
       img.height = targetHeight
 
+    # Return the callback if images already fit, and resizeUp is false
+    imagesWidth = imgsWidth()
```
Comment (craigspaeth) [inline index.coffee:46]: This is probably cheap enough to not have to store the result in a variable, but 👍  works for me.

**Comment by craigspaeth** [comment]
Solid thanks!

File: index.coffee [hunk 2]
```diff
@@ -11,14 +11,15 @@ jqueryFillwidthLite = (a, b, c) ->
       $(this),
       (options.targetHeight or 500),
       (options.done or ->),
-      (options.apply or defaultApply)
-      (options.gutterSize or 0)
+      (options.apply or defaultApply),
+      (options.gutterSize or 0),
+      (options.resizeUp or false)
```
Comment (craigspaeth) [inline index.coffee:16]: Shoot, this should default to `true` for backwards compatibility.

Comment (mzikherman) [inline index.coffee:16]: oh yea of course, good point.

**Pull request merged**